src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -183,6 +183,26 @@ public UserTypeClassesFormatDefinition()
             }
         }
         #endregion
+        #region User Types - Records
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.RecordName)]
+        [Name(ClassificationTypeNames.RecordName)]
+        [Order(After = PredefinedClassificationTypeNames.Identifier)]
+        [Order(After = PredefinedClassificationTypeNames.Keyword)]
+        [Order(Before = ClassificationTypeNames.StaticSymbol)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
+        private class UserTypeRecordsFormatDefinition : ClassificationFormatDefinition
+        {
+            [ImportingConstructor]
+            [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+            public UserTypeRecordsFormatDefinition()
+            {
+                this.DisplayName = EditorFeaturesResources.User_Types_Records;
+                this.ForegroundColor = Color.FromRgb(0x2B, 0x91, 0xAF);
+            }
+        }
```

Comment:
i assume this is the color for classes.  i"m on the fence if i want a different color here :)  might be fun to call out records as the snazzy newnews.  @mikadumont for thoughts.  We can possibly pick one of the safe colors we found for regexes.    that would a different PR though :)

Reply:
@JoeRobich Do you need to do anything for advanced classification for this?

Reply:
question still applies.  @JoeRobich

Reply:
@CyrusNajmabadi What do you mean by advanced classification?

Reply:
whatever this stuff is:

![image](https://user-images.githubusercontent.com/4564579/95400454-7d5db100-08bf-11eb-89ce-18f856595531.png)


:)

Reply:
Nope. Since we are inheriting the ClassName color, there is no need to add entries to the ColorScheme files. When we decide a custom color is required we can add the entries at that time.